### PR TITLE
[BUG] Some NPEs from Google Play Console

### PR DESCRIPTION
**Pull request by Aitorbp**
## Related Issues
App: https://github.com/owncloud/android/issues/4207

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java
Comment (JuancaG05) [inline owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java:618]: Not sure if the NPE refers to this line or something inside the `refreshFolderUseCase` method... `mReceiveExternalFilesViewModel` shouldn't be `null` because it's obtained by Koin dependency injection in a previous executed method 🤔

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt
Comment (JuancaG05) [inline owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt:1626]: Looking at the report, it seems that the same is happening in the line 1642, where `storageManager` is used as well

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Some things to check here @Aitorbp, maybe a little research on what would happen if those variables are `null` and, now that we handle nullability, we won't get their results? 🤔

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java
Comment (Aitorbp) [reply owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java:618]: Yes, true, but with the information we have in the trace we do not know exactly what is causing the null. Since we were also unable to reproduce the error, I would opt for this approach.

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt
Comment (Aitorbp) [reply owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt:1626]: In relation to the first case of error. We are going to do the following approach: In the `getStorageManager` getter of the `BaseActivity`, we are going to put a condition whether `mStorageManager` is `null`. In this case we will create a new instance of `FileDataStorageManager` using the current account.

**Review by JuancaG05 — APPROVED** [review]
LGTM, let's see if this works now!

**Comment by jesmrec** [comment]
I did some checks regarding the FileDataStorageManager and the ReceiveExternalFileActivity to check if nullability checks affect somehow. For me, we can move forward this one, waiting for the next release to check whether crash reports are not there anymore.

Moving this forward

**Pull request merged**
